media/js/firefox/new/install/experiment.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function(Mozilla) {
+    'use strict';
+
+    var isChrome = !!window.chrome;
```

Comment:
The experiment is activating on Edge in windows for me. `window.chrome` returns an object.

Reply:
Crap, great catch Stephanie! Let me try and find some horrendous piece of code to fix this 😕